zenodo/modules/records/data/objecttypes.json

```diff
@@ -602,4 +604,44 @@
     "type": "publication"
   },
   "csl": "graphic"
+},
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "publication-datamanagementplan",
+  "id": "https://zenodo.org/objecttypes/publication/datamanagementplan#",
+  "title": {
+    "en": "Data management plan"
+  },
+  "title_plural": {
+    "en": "Data management plans"
+  },
+  "schema.org": "https://schema.org/CreativeWork",
+  "datacite": {"general": "Text", "type": "Data Management Plan"},
+  "eurepo": "info:eu-repo/semantics/technicalDocumentation",
+  "parent": {"$ref": "https://zenodo.org/objecttypes/publication"},
+  "openaire": {
+    "resourceType": "0009",
+    "type": "publication"
+  },
+  "csl": "report"
```

Comment:
@lnielsen WDYT? Available types are [here](https://github.com/citation-style-language/documentation/blob/2f6161950734da35fe0538ed5b6f6da0be260136/specification.txt#L2517-L2551) (also see below for "Annotation collection")